school_people.rb

```diff
@@ -0,0 +1,7 @@
+module School_Person
```

Comment:
This name is fine for a class (though it should not have the underscore), but a module should be "<something>able". In this case... `HighFiveable`